Tests/APITesters/.swiftlint.yml

```diff
@@ -2,3 +2,4 @@ disabled_rules:
   - file_length
   - type_body_length
   - function_body_length
+  - force_try
```

Comment:
nit: Not an expert here, but only seeing one `try!` in this PR, so is this worth trying to leave enabled?
My research took me to https://realm.github.io/SwiftLint/force_try.html and https://cocoacasts.com/what-is-the-difference-between-try-try-and-try where it looks like they spell out good reasons to avoid `try!`.. so thought I would raise it.

Reply:
Oh I forgot to explain this. We don't actually _run_ APITester code, we only compile it (thats by design). `try!` would crash at runtime, but because it never runs it doesn't matter. It simplifies the type checking.